starlight_help/src/content/include/_ChannelFoldersIntro.mdx

```diff
@@ -7,3 +7,6 @@ whether to group them by folder in the [left sidebar](/help/left-sidebar).
 Everyone can [pin channels](/help/pin-a-channel) they personally want to pay
 close attention to. Pinned channels appear in a dedicated **pinned channels**
 section above channel folders.
+
+Folders that only contain [inactive](/help/manage-inactive-channels) and
+[muted](/help/mute-a-channel) channels are sorted at the bottom.
```

Comment:
We need to update the language here around grouping in the inbox, now that we've merged that feature. 

And there's probably then a follow-up to make the grouping-at-bottom behavior implemented here work in inbox, too?

Reply:
OK, maybe that's right regarding inbox. And I think @laurynmm already did a aba4228e931c90af98ec87735bb02de0aa711d17 for the documentation I mentioned.